src/swell/utilities/data_assimilation_window_params.py

```diff
@@ -33,14 +33,14 @@ def __init__(self, logger, cycle_time):
 
     # ----------------------------------------------------------------------------------------------
 
-    def __get_window_begin_dto__(self, window_offset):
+    def __get_window_begin_dto__(self, window_offset: str) -> datetime:
```

Comment:
`datetime` is the module; the class is `datetime.datetime`.

```suggestion
    def __get_window_begin_dto__(self, window_offset: str) -> datetime.datetime:
```